### DISCUSSION TITLE
有在Android或iOS端使用的例子吗？

### DISCUSSION THREAD

**Discussion by szuwest** [fluttercandies/baidupan#3] [Q&A]
### Content

看上去像是支持Android和iOS端的，但是没有找到文档或example

**Comment by CaiJingLong** [comment]
纯 dart 库，没有特地为安卓或iOS做兼容
但是理论上是支持的
目前的问题在于没有针对 flutter 端做优化，所以获取 access_token 的过程对于手机用户并不友好
